partition/roles/monitoring/README.md

```diff
@@ -0,0 +1,45 @@
+# monitoring
```

Comment:
As this folder is not a role but only contains roles, I think we should put individual README.md files into the folders which are actually roles.